### PR TITLE
Change "plug-in" to "plugin"

### PR DESCRIPTION
**Pull request by apinnick**
Change "plug-in" to "plugin" to align with [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#_p).

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_acd-developers.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_acd-developers.adoc:1]: This file has been dropped with 48ad6a1daf4285430ca3a96f78a350f753fd0009 > please rebase your branch to HEAD of "master".

File: guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 Create a simple shellhook script that prints `Hello World!` when you run a remote execution job.
 
-.Prerequisites
-* You have the webhooks and shellhooks plug-ins installed.
+.Prerequisite
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc:6]: We have also changed this to plural globally. The diff should go away when you rebase.

File: guides/common/modules/proc_installing-amazon-ec2-plugin.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
-[id="Installing_Amazon_EC2_Plug-in_{context}"]
-= Installing Amazon EC2 plug-in
+[id="Installing_Amazon_EC2_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-amazon-ec2-plugin.adoc:1]:
```suggestion
[id="Installing_Amazon_EC2_plugin_{context}"]
```

File: guides/common/modules/proc_installing-google-gce-plugin.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
-[id="Installing_Google_GCE_Plug-in_{context}"]
-= Installing Google GCE plug-in
+[id="Installing_Google_GCE_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-google-gce-plugin.adoc:1]:
```suggestion
[id="Installing_Google_GCE_plugin_{context}"]
```

File: guides/common/modules/proc_installing-microsoft-azure-plugin.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
-[id="Installing_Microsoft_Azure_Plug-in_{context}"]
-= Installing Microsoft Azure plug-in
+[id="Installing_Microsoft_Azure_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-microsoft-azure-plugin.adoc:1]:
```suggestion
[id="Installing_Microsoft_Azure_plugin_{context}"]
```

File: guides/common/modules/proc_live-patching-hosts-using-kernelcare.adoc [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [id="Live_Patching_Hosts_Using_KernelCare_Plug_In_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_live-patching-hosts-using-kernelcare.adoc:1]:
```suggestion
[id="Live_Patching_Hosts_Using_KernelCare_plugin_{context}"]
```

File: guides/common/modules/proc_installing-vmware-plugin.adoc [hunk 6]
```diff
@@ -1,7 +1,7 @@
-[id="Installing_VMware_Plug-in_{context}"]
-= Installing the VMware plug-in
+[id="Installing_VMware_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-vmware-plugin.adoc:1]:
```suggestion
[id="Installing_VMware_plugin_{context}"]
```

File: guides/common/modules/proc_installing-the-snapshot-management-plugin.adoc [hunk 7]
```diff
@@ -1,10 +1,10 @@
-[id="Installing_the_Snapshot_Management_Plug-in_{context}"]
-= Installing the Snapshot Management plug-in
+[id="Installing_the_Snapshot_Management_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-snapshot-management-plugin.adoc:1]:
```suggestion
[id="Installing_the_Snapshot_Management_plugin_{context}"]
```

File: guides/common/modules/proc_installing-the-scc-manager.adoc [hunk 8]
```diff
@@ -1,10 +1,10 @@
-[id="Installing_the_SCC_Manager_Plug-in_{context}"]
-= Installing the SCC Manager plug-in
+[id="Installing_the_SCC_Manager_Plugin_{context}"]
+= Installing the SCC Manager plugin
 
-Use this procedure to install the _SCC Manager_ plug-in on your {Project}.
+Use this procedure to install the _SCC Manager_ plugin on your {Project}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-scc-manager.adoc:4]:
```suggestion
Use this procedure to install the SCC Manager plugin on your {Project}.
```

File: guides/common/modules/proc_installing-the-scc-manager.adoc [hunk 9]
```diff
@@ -1,10 +1,10 @@
-[id="Installing_the_SCC_Manager_Plug-in_{context}"]
-= Installing the SCC Manager plug-in
+[id="Installing_the_SCC_Manager_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-scc-manager.adoc:1]:
```suggestion
[id="Installing_the_SCC_Manager_plugin_{context}"]
```

File: guides/common/modules/proc_installing-the-salt-plugin.adoc [hunk 10]
```diff
@@ -1,18 +1,18 @@
-[id="Installing_the_Salt_Plug-in_{context}"]
-= Installing the Salt plug-in
+[id="Installing_the_Salt_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-salt-plugin.adoc:1]:
```suggestion
[id="Installing_the_Salt_plugin_{context}"]
```

File: guides/common/modules/proc_installing-shellhooks-plugin.adoc [hunk 11]
```diff
@@ -1,8 +1,12 @@
-[id="installing-shellhooks-plug-in_{context}"]
-= Installing the shellhooks plug-in
+[id="installing-shellhooks-plugin_{context}"]
+= Installing the shellhooks plugin
 
-Optionally, you can install and enable the shellhooks plug-in on each {SmartProxy} used for shellhooks, using the following command:
+Optionally, you can install and enable the shellhooks plugin on each {SmartProxy} used for shellhooks.
 
+.Procedure
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-shellhooks-plugin.adoc:7]:
```suggestion
```

File: guides/common/modules/proc_installing-red-hat-cloud-plugin.adoc [hunk 12]
```diff
@@ -0,0 +1,13 @@
+[id="Installing_Red_Hat_Cloud_Plugin_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-red-hat-cloud-plugin.adoc:1]:
```suggestion
[id="Installing_Red_Hat_Cloud_plugin_{context}"]
```

File: guides/common/modules/proc_creating-snapshots-of-a-host.adoc [hunk 13]
```diff
@@ -1,19 +1,19 @@
...
 = Creating snapshots of a host
 
-You can use the Snapshot Management plug-in to create snapshots of hosts.
+You can use the Snapshot Management plugin to create snapshots of hosts.
 
 .Prerequisites
-* You have installed the Snapshot Management plug-in successfully.
-For more information, see xref:Installing_the_Snapshot_Management_Plug-in_{context}[].
+* You have installed the Snapshot Management plugin successfully.
+For more information, see xref:Installing_the_Snapshot_Management_Plugin_{context}[].
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-snapshots-of-a-host.adoc:8]:
```suggestion
For more information, see xref:Installing_the_Snapshot_Management_plugin_{context}[].
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_acd-developers.adoc
Comment (apinnick) [reply guides/common/modules/proc_acd-developers.adoc:1]: Whoops! Thought I had. Will do it now.

File: guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc (hunk 1 above)
Comment (apinnick) [reply guides/common/modules/proc_creating-a-shellhook-to-print-arguments.adoc:6]: Looks like it was missed. Hope it's OK if I fix it in this PR.

File: guides/common/modules/proc_acd-developers.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_acd-developers.adoc:1]: This has been deleted before.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 14]
```diff
@@ -3,14 +3,23 @@
 
 A sub-status monitors only a part of a host's capabilities.
 
-To view the sub-statuses of a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click on the host whose full status you want to inspect.
-You can view the global host status next to the name of the host and the host sub-statuses on the *Host status* card.
+Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc:6]: still a bigger diff than expected.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
I think rebasing/merging introduced two minor issues; please have a look at them again. rest LGTM

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 14 above)
[the remark this answers is not in the document]
Comment (apinnick) [reply guides/common/modules/con_host-substatus-overview.adoc:6]: I don't know how this got into my PR. Will fix.

**Comment by apinnick** [comment]
> 111/112 LGTM; guides/common/modules/proc_acd-developers.adoc still there

@maximiliankolb "The file that refused to die". I drove a wooden stake into its heart. I think it's gone for good. :-)

**Review by maximiliankolb — APPROVED** [review]
Thanks Avital, LGTM. Can someone else please ACK this too? I remember that we've switched from "plugin" to "plug-in" in the past; so I'd like an additional "go". I will merge with a second approval.

**Comment by apinnick** [comment]
> Thanks Avital, LGTM. Can someone else please ACK this too? I remember that we've switched from "plugin" to "plug-in" in the past; so I'd like an additional "go". I will merge with a second approval.

@maximiliankolb  What you said just rang a bell, so I went to check all our style guides. RHSSG says "plugin" while ISG (IBM Style Guide, which we use d/s) says to use "plug-in". Normally, RHSSG takes precedence over ISG, but there is usually a note. Please hold off on merging this while I check with someone who works on the RHSSG. Will get back to you.

**Comment by apinnick** [comment]
> > Thanks Avital, LGTM. Can someone else please ACK this too? I remember that we've switched from "plugin" to "plug-in" in the past; so I'd like an additional "go". I will merge with a second approval.
> 
> @maximiliankolb What you said just rang a bell, so I went to check all our style guides. RHSSG says "plugin" while ISG (IBM Style Guide, which we use d/s) says to use "plug-in". Normally, RHSSG takes precedence over ISG, but there is usually a note. Please hold off on merging this while I check with someone who works on the RHSSG. Will get back to you.

@maximiliankolb  I found the [PR](https://github.com/redhat-documentation/supplementary-style-guide/pull/253) that changed 'plug-in' to 'plugin' for RHSSG (closes [issue 213](https://github.com/redhat-documentation/supplementary-style-guide/issues/213). Looks like the "caution" icon is used for the 'old' (IBM) usage and the "checkmark" for the 'new' (RH) usage. So it's OK to change the spelling to "plugin".

**Review by asteflova — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   c821b2b0a7..4c6ebc700f  3.10 -> 3.10
   20e81e6371..37e5f16707  3.9 -> 3.9

**Pull request merged**